app/services/workers/process-import.js

```diff
@@ -22,11 +23,12 @@ module.exports.execute = function (task) {
   const batchSize = parseInt(config.ASYNC_WORKER_BATCH_SIZE, 10)
   const tasks = []
   let workloadReportId
-
   return insertWorkloadReport()
-    .then(function (insertedWorkloadReportId) {
-      workloadReportId = insertedWorkloadReportId
-      return populateStagingCourtReporters()
+    .then(function (insertedWorkloadReport) {
+      workloadReportId = insertedWorkloadReport.id
+      return updateCloseOpenWorkloadReports(insertedWorkloadReport.effective_from).then(function () {
```

Comment:
I think this will close the workload report we have just inserted as well as the previous one